index.html

```diff
@@ -16,44 +16,71 @@
   <meta name="generator" content="Polymer Starter Kit">
   <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
-  <title>My App</title>
-  <meta name="description" content="My App description">
+  <title>Request Yo Racks</title>
+  <meta name="description" content="Request Yo Rack">
 
   <!--
       If deploying to a non-root path, replace href="/" with the full path to the project root.
       For example: href="/polymer-starter-kit/relative-path-example/"
     -->
   <base href="/">
 
-  <link rel="icon" href="images/favicon.ico">
-
-  <!-- See https://goo.gl/OOhYW5 -->
+  <link rel="icon" href="images/manifest/favicon.ico">
+  <link rel="icon" type="image/png" sizes="16x16" href="images/manifest/favicon-16x16.png">  
+  <link rel="icon" type="image/png" sizes="32x32" href="images/manifest/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="96x96" href="images/manifest/favicon-96x96.png">
   <link rel="manifest" href="manifest.json">
+  <meta name="theme-color" content="#ffffff">
+
+  <!-- Android Icons -->
+  <link rel="apple-touch-icon" sizes="36x36" href="images/manifest/android-icon-36x36.png">
```

Comment:
There is no need to have platform specific icons for android/apple/ms. They should all point to the `images/logos` directory, and you should be able to remove the images from `images/manifest`. 

If some icon sizes are missing you may add them in `images/logos`. Check with @2echoi if you need further help with the logos.